### ISSUE TITLE
Stop validation process on "remove token"

### ISSUE THREAD

**Issue by jcramer** [simpleledger/Electron-Cash-SLP#87]
When a token is in the process of validating and then it is removed from the tokens tab we should trigger the validation job to exit.

**Comment by cculianu** [comment]
Good call. Will see if can be easily added -- it should not be too bad.

**Comment by jcramer** [comment]
I took a look at implementing this.  

When a token is removed from a wallet, the validation jobs for that token ID and _that_ wallet should be stopped.  With the current organization, I'm not seeing a good way to stop jobs based on the wallet.  Also if the user has two wallets open that are both depending on the same validation job, there doesn't appear to be a way to know that.

I'm going to change this from `bug` to `enhancement`, because its not really breaking anything or preventing anything from working.
